code/datums/status_effects/blob_burst.dm

```diff
@@ -0,0 +1,27 @@
+/datum/status_effect/blob_burst
+	alert_type = /obj/screen/alert/status_effect/blob_burst
+	var/datum/callback/blob_burst_callback
+
+/datum/status_effect/blob_burst/on_creation(mob/living/new_owner, duration = 120 SECONDS, datum/callback/burst_callback)
+	src.duration = duration
+	. = ..()
+	if(!.)
+		return
+	blob_burst_callback = burst_callback
+
+/datum/status_effect/blob_burst/Destroy()
+	blob_burst_callback = null
```

Comment:
Qdel null?

Reply:
> Qdel null?

Shouldn't be required here due to the callback only being saved here, nor does the callback itself have any `Destroy` logic to clean up its variables. By just setting it to null it will be cleaned up since all references got cleared, thus also clearing the references the callback had.